src/ports/items/utils.ts

```diff
@@ -316,9 +319,19 @@ export function getItemsQuery(filters: ItemFilters, isCount = false) {
 }
 
 /**
- * It returns the minimum sale value allowed for item sales.
+ * It returns the minimum sale value allowed for item sales if any.
  * The price is expected to be used as an inclusive cap, meaning that prices that equal the value SHOULD be filtered
  */
-function getMinSaleValueInWei(): string {
-  return process.env.MIN_SALE_VALUE_IN_WEI || '0'
+function getMinSaleValueInWei(): string | undefined {
+  return process.env.MIN_SALE_VALUE_IN_WEI
+}
+
+function isFragmentOnSale(fragment: ItemFragment) {
+  const minSaleValue = getMinSaleValueInWei()
+
+  const isOnSale = fragment.searchIsStoreMinter && +fragment.available > 0
+  const isValidSalePrice =
+    !minSaleValue || new BN(fragment.price).gte(new BN(minSaleValue))
```

Comment:
I see that the isOnSale value before had this logic but with .lte instead of .gte. This one is the correct one right?

Reply:
think so, yes! we arrived at:

`if (price < min) wei then throw`

so this should be the opposite